app/assets/scripts/components/explore/Summary.js

```diff
@@ -83,6 +83,13 @@ class Summary extends Component {
             <p className='exp-summary__subtitle'>
               Results for {this.props.appliedState.year}
             </p>
+            {model.disclaimer &&
+              <div className='exp-summary__disclaimer'>
+                <p>
```

Comment:
Since the disclaimer is always going to be just a paragraph, could we lose the `div`? (Like subtitle above)